training_content/wiki_ed/slides/60-using-the-dashboard/6002-add-peer-review-articles.yml

```diff
@@ -0,0 +1,14 @@
+id: 6002
+title: "Select articles to peer review"
+summary:
+content: |
+  Click "Review an article" to see the topics your classmates are working on and choose
+  which ones you will peer review.
+
+  When you select a classmate's article to review, the listing will include links to their
+  bibliography and draft pages, a link to the page where you should complete your peer review,
+  and a link to the current version of the article (if it's already live on Wikipedia).
```

Comment:
Is it worthwhile to add something in here about how the links are the _suggested_ places for doing that work?

Reply:
I think the closer we get to 'this is the place, period', the better it will work.